docs/sources/reference/api/registry_api_client_libraries.md

```diff
@@ -2,7 +2,7 @@ page_title: Registry API Client Libraries
 page_description: Various client libraries available to use with the Docker registry API
 page_keywords: API, Docker, index, registry, REST, documentation, clients, C#, Erlang, Go, Groovy, Java, JavaScript, Perl, PHP, Python, Ruby, Rust, Scala
 
-# Docker Registry API Client Libraries
+# Docker Registry 1.0 API Client Libraries
```

Comment:
The Registry is at 1.0 I have no idea where these client libraries are at version wise. We don't own those projects.